stoc.py

```diff
@@ -34,9 +35,27 @@ def toc(self):
         markdown_toc = ""
         for title_size, title in self.toc_items:
             h = int(title_size.replace("h", ""))
-            markdown_toc += " " * 2 * h + "- " + f'<a href="#{normalize(title)}" class="toc"> {title}</a> \n'
+            markdown_toc += (
+                " " * 2 * h
+                + "- "
+                + f'<a href="#{normalize(title)}" class="toc"> {title}</a> \n'
+            )
```

Comment:
Better! Some day we'll just use https://github.com/tvst/htbuilder everywhere instead of these raw strings

Reply:
Or we could use https://pypi.org/project/py-emmet/